.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-  "semi": true,
+  "semi": false,
```

Comment:
Hi @Alexander-Sibhatu, why don't we want semicolons? This will introduce inconsistency between ESLint and Prettier, which could lead to errors. Can we please set it to true again? 
cc: @mcromp